package.json

```diff
@@ -15,7 +15,8 @@
     "prebuild": "ts-node -O '{\"module\":\"commonjs\"}' git.version.ts && ./scripts/generate-openapi-script.sh",
     "prebuild.prod": "npm run prebuild",
     "build.prod": "npx ng build --prod --subresource-integrity",
```

Comment:
I thought you [said](https://github.com/dockstore/dockstore-ui2/pull/1325#issuecomment-897138979) you were changing this to `-c production` in your PR, or did you mean another PR?

Reply:
oh, i only changed the "build"